api/autopilot.go

```diff
@@ -137,8 +129,8 @@ type (
 	}
 )
 
-func (ap Autopilot) EndHeight() uint64 {
-	return ap.CurrentPeriod + ap.Contracts.Period + ap.Contracts.RenewWindow
+func (ap AutopilotConfig) EndHeight() uint64 {
+	return ap.Contracts.Period + ap.Contracts.RenewWindow
```

Comment:
The math here is incorrect now since it will always return the same end height for a given config